### ISSUE TITLE
Error Upon Startup

### ISSUE THREAD

**Issue by Storm-Trooper5555** [Opticos/GWSL-Source#180]
![Screenshot (16)](https://github.com/Opticos/GWSL-Source/assets/157601224/3039bba0-db80-4699-8050-a39fecb643e6)
Hi there

Has anyone else gotten this issue and/or fixed it because I seem to keep getting this error when I try to run the GWSL.exe file in the dist\GWSL_145 MSIX build 3\ directory  after I have built it with no errors displayed in the output with either of the releases.

(I did try editing the manager.py script and declaring variables as global but it would just move on to the next variable with the exact same error but different variables)

I tried with Python 3.7 and Python 3.12 and got the same result for both.

Thanks

**Comment by wowitsjack** [comment]
> ![Screenshot (16)](https://private-user-images.githubusercontent.com/157601224/299315283-3039bba0-db80-4699-8050-a39fecb643e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcyMDA3MDMsIm5iZiI6MTcwNzIwMDQwMywicGF0aCI6Ii8xNTc2MDEyMjQvMjk5MzE1MjgzLTMwMzliYmEwLWRiODAtNDY5OS04MDUwLWEzOWZlY2I2NDNlNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwNlQwNjIwMDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YjlhYWQxNmJjNzRkOTRiYTRkNDFlMmVkNThkMmVhZTlmYmQ5NWViM2EyYzYwMjg5Njg2MDI5ODFlZWI2ODRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.w6HPyDHZ2FtyfRleWfe71Ttz7kGUq4efa2fvvVBt4ZM) Hi there
> 
> Has anyone else gotten this issue and/or fixed it because I seem to keep getting this error when I try to run the GWSL.exe file in the dist\GWSL_145 MSIX build 3\ directory after I have built it with no errors displayed in the output with either of the releases.
> 
> (I did try editing the manager.py script and declaring variables as global but it would just move on to the next variable with the exact same error but different variables)
> 
> I tried with Python 3.7 and Python 3.12 and got the same result for both.
> 
> Thanks

No. It looks like the author has given up on the idea of this being open source. 

Dev's constantly updating the paid Windows store version, and deliberately holds back any free binaries, and the source code provided doesn't build.

**Comment by PaxtonPenguin** [comment]
well why even show the source code anymore?

**Comment by GPopZach** [comment]
Also getting this issue, haven't found a fix yet.

**Comment by Pololot64** [comment]
I would help... But none of these posts provide any info. "The app doesn't run" is very vague.

**Comment by KMohZaid** [comment]
> > ![Screenshot (16)](https://private-user-images.githubusercontent.com/157601224/299315283-3039bba0-db80-4699-8050-a39fecb643e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcyMDA3MDMsIm5iZiI6MTcwNzIwMDQwMywicGF0aCI6Ii8xNTc2MDEyMjQvMjk5MzE1MjgzLTMwMzliYmEwLWRiODAtNDY5OS04MDUwLWEzOWZlY2I2NDNlNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwNlQwNjIwMDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YjlhYWQxNmJjNzRkOTRiYTRkNDFlMmVkNThkMmVhZTlmYmQ5NWViM2EyYzYwMjg5Njg2MDI5ODFlZWI2ODRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.w6HPyDHZ2FtyfRleWfe71Ttz7kGUq4efa2fvvVBt4ZM) Hi there
> > Has anyone else gotten this issue and/or fixed it because I seem to keep getting this error when I try to run the GWSL.exe file in the dist\GWSL_145 MSIX build 3\ directory after I have built it with no errors displayed in the output with either of the releases.
> > (I did try editing the manager.py script and declaring variables as global but it would just move on to the next variable with the exact same error but different variables)
> > I tried with Python 3.7 and Python 3.12 and got the same result for both.
> > Thanks
> 
> No. It looks like the author has given up on the idea of this being open source.
> 
> Dev's constantly updating the paid Windows store version, and deliberately holds back any free binaries, and the source code provided doesn't build.
also refering to : https://github.com/Opticos/GWSL-Source/issues/182
hey i dont think that the case is 

i just saw that microsoft store version is 1.4.5 https://apps.microsoft.com/detail/9nl6kd1h33v3?hl=en-us&gl=US and last release changelog is of same version https://github.com/Opticos/GWSL-Source/releases/tag/v145


I just build it manually , https://github.com/KMohZaid/GWSL-Source/actions/runs/10047113174

BUT

issues :
1. GWSL_service.exe  gives error "Failed to execute script main", GWSL.exe stop when i run it

2. If i run main.py which is service, it lauches vnc and audio pulse runner, but still when i run manager.py it says error can't start service. exporting display throws some error due to miss format in code
    -  this tells us why GWSL.exe not running. It tries to run service file but service file got error

I think it have compatible issue as of now or maybe problem? i will try to run it later if it works i will tell you

**Comment by KMohZaid** [comment]
PS : i think it is possible to revive this source code, goodnight for now...

**Comment by Pololot64** [comment]
It is pretty bizarre that it doesn't build. It should. And the source code is more recent than the last release btw. I have a feeling that the build dependencies must be incorrect. Lmk when you try again.

**Comment by KMohZaid** [comment]
> It is pretty bizarre that it doesn't build. It should. And the source code is more recent than the last release btw. I have a feeling that the build dependencies must be incorrect. Lmk when you try again.

ok, i think it is because of some other issue

idk how they were able to test it without building (seeing building logs from action it seems like they build service then tested manager.py bcz code try to launch that file)

i will maybe start working on it after 12pm IST if rain not interrupt

**Comment by Pololot64** [comment]
Feel free. Btw to everyone curious, I do plan to make changes to GWSL as needed. I am currently a college student with a job and do not have much time on my hands any more so that's why it takes time.

**Comment by KMohZaid** [comment]
> Feel free. Btw to everyone curious, I do plan to make changes to GWSL as needed. I am currently a college student with a job and do not have much time on my hands any more so that's why it takes time.

ok thanks, i was able ot run it properly, now fixing start issue

**Comment by KMohZaid** [comment]
> Feel free. Btw to everyone curious, I do plan to make changes to GWSL as needed. I am currently a college student with a job and do not have much time on my hands any more so that's why it takes time.

ongoing PR is opened : https://github.com/Opticos/GWSL-Source/pull/186
lets discussion fixing suff there
